enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -351,295 +344,180 @@ web.xml or a partial web.xml:
-SEI but is not required to do so. If SEI is not specified, the Service
-Implementation Bean class implicitly defines a SEI as required by
-section 3.3 of JAX-WS specification. The business methods of the bean
-must be public and must not be final or static. Only those methods that
-are annotated with @WebMethod in the Service Implementation Bean, are
-exposed to the client.
-* For developers starting from WSDL using JAX-WS, the SEI generated from
-the WSDL must be annotated with javax.jws.WebService annotation. Refer
-to section 2.2 of JAX-WS specification. The Service Implementation Bean
-must be annotated with javax.jws.WebService annotation and the
-endpointInterface member attribute must refer to this generated SEI.
-Service Implementation Bean may implement the Service Endpoint
-Interface, but it is not required to do so. The bean must implement all
-the method signatures of the SEI. The business methods of the bean must
-be public and must not be final or static. It may implement other
-methods in addition to those defined by the SEI.
+jakarta.jws.WebService or jakarta.xml.ws.WebServiceProvider annotation. 
+* For developers starting from Java, jakarta.jws.WebService annotation 
+on Service Implementation Bean may optionally reference an SEI but is 
+not required to do so. If SEI is not specified, the Service Implementation 
+Bean class implicitly defines a SEI as required by Jakarta XML Web Services 
+specification. The business methods of the bean must be public and must not 
+be final or static. Only those methods that are annotated with @WebMethod 
+in the Service Implementation Bean, are exposed to the client.
+* For developers starting from WSDL, the SEI generated from the WSDL must 
+be annotated with jakarta.jws.WebService annotation. Refer to Jakarta XML 
+Web Services specification. The Service Implementation Bean must be annotated 
+with jakarta.jws.WebService annotation and the endpointInterface member 
+attribute must refer to this generated SEI.  Service Implementation Bean may 
+implement the Service Endpoint Interface, but it is not required to do so. 
+The bean must implement all the method signatures of the SEI. The business 
+methods of the bean must be public and must not be final or static. 
+It may implement other methods in addition to those defined by the SEI.
 * The Service Implementation Bean must have a default public
 constructor.
 
 * A Service Implementation must be a stateless object. A Service
 Implementation Bean must not save client specific state across method
-calls either within the bean instance’s data members or external to the
+calls either within the bean instance's data members or external to the
 instance. A container may use any bean instance to service a request.
 * The class must be public, must not be final and must not be abstract.
 * The class must not define the finalize() method.
 
 ======= The optional @PostConstruct or @PreDestroy annotations
```

Comment:
and this can be moved one level up, so the formatting is not broken...

Reply:
yes, will remove the extra line